debian/control

```diff
@@ -13,7 +13,7 @@ Build-Depends: automake,
                cpio,
                cryptsetup-bin | cryptsetup,
                cython3,
-               debhelper (>= 9),
+               debhelper (>= 10) | dh-systemd,
```

Comment:
IIUC, `dh-systemd` is now only a transitional package. and we can assume that debhelper offers the functionality of `dh-system`, can we just drop it? like
```
debhelper (>= 10),
```
?

Reply:
I think so?  Yes.  I ran across https://github.com/callahad/synapse/commit/c71c1ede534153f71c96383b291f84fafed6a57e but it's only written as `debhelper (>= 10) | dh-systemd,` to keep support for Xenial which we've dropped.

Will fix again.